Mathlib/Analysis/NormedSpace/Multilinear.lean

```diff
@@ -330,25 +330,19 @@ theorem bounds_bddBelow {f : ContinuousMultilinearMap 𝕜 E G} :
   ⟨0, fun _ ⟨hn, _⟩ => hn⟩
 #align continuous_multilinear_map.bounds_bdd_below ContinuousMultilinearMap.bounds_bddBelow
 
+theorem isLeast_op_norm : IsLeast {c : ℝ | 0 ≤ c ∧ ∀ m, ‖f m‖ ≤ c * ∏ i, ‖m i‖} ‖f‖ := by
```

Comment:
Can you add the `nnnorm` version too, which has a slightly nicer statement? I don't have any feel for whether one is easy to prove from the other.